docs/gallery/interactive/sofar_introduction.ipynb

```diff
@@ -0,0 +1,369 @@
+    "* **Double Variables:**\n",
+    "    Variables of type *double* store numeric data and can be entered as numbers, lists, or numpy arrays.\n",
+    "* **String Variables:**\n",
+    "    Variables of type *string* store strings and can be entered as strings, lists of string, or numpy string arrays.\n",
+    "\n",
+    "The data can be mandatory, optional, and read only and must have a shape (dimension in SOFA language) according to the underlying convention. Read on for more information.\n",
+    "\n",
+    "To get a quick insight into SOFA objects\n",
+    "\n",
+    "* ``sofa.inspect`` prints the data stored in a SOFA object or at least gives\n",
+    "  the shape in case of large arrays that would clutter the output. This is\n",
+    "  helpful when reading data from an existing SOFA object.\n",
+    "* ``sofa.list_dimensions`` prints the dimensions of the data inside the SOFA\n",
+    "  object.\n",
+    "* ``sofa.get_dimension`` returns the size of a specific dimension.\n",
+    "\n",
+    "For the *SimpleFreeFieldHRIR* SOFA object we have the following dimensions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sofa.list_dimensions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case, `M` denotes the number of source positions for which HRIRs are available, `R` is the number of ears - which is two - and `N` gives the lengths of the HRIRs in samples. `S` is zero, because the convention does not have any string variables. `C` is always three, because coordinates are either given by x, y, and z values or by their azimuth, elevation and radius in degree."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is important to be aware of the dimensions and enter data as determined by the convention. SOFA sets the `dimensions` implicitly. This means the dimensions are derived from the data itself, as indicated by the output of `sofa.list_dimensions` above (*set by...*). In some cases, variables can have different shapes. An example for this is the `ReceiverPosition` which can be of shape RCI or RCM. To get a dimension as a variable use"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sofa.get_dimension('N')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's assume you want to have a better overview of the data contained in the Sofa object. For this purpose you can use `sofa.inspect`. This lists the information contained in the SOFA attributes, and gives at least the shapes of all SOFA variables. In case a variable contains only a few entries, its values will be shown as well. This will most likely give you a better idea of the data then\n",
```

Comment:
nice, this was what i was looking for, i think we can do this earlier, as mentions before. It would help to understand was is going on and when you describe the attributes.

Reply:
yes, moved up!